cmd/git-sync/main.go

```diff
@@ -625,21 +625,21 @@ func runCommand(ctx context.Context, cwd, command string, args ...string) (strin
 
 func setupGitAuth(username, password, gitURL string) error {
 	log.V(1).Info("setting up git credential cache")
-	cmd := exec.Command(*flGitCmd, "config", "--global", "credential.helper", "cache")
-	output, err := cmd.CombinedOutput()
+
+	_, err := runCommand(context.TODO(), "", *flGitCmd, "config", "--global", "credential.helper", "cache")
 	if err != nil {
-		return fmt.Errorf("error setting up git credentials %v: %s", err, string(output))
+		return fmt.Errorf("error setting up git credentials: %v", err)
 	}
 
-	cmd = exec.Command(*flGitCmd, "credential", "approve")
+	cmd := exec.Command(*flGitCmd, "credential", "approve")
```

Comment:
Can you please add a comment where that this is not using `runCommand` because we need to fiddle with stdin ?

Reply:
Since I've added `runCommandWithStdin` this is not needed anymore